### PR TITLE
NuGet.exe now has a push option called SkipDuplicate

### PR DESCRIPTION
**Pull request by donnie-msft**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[29addfa](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1444/commits/29addfaf5fa05f6bc07e1b058f2358de2a713268)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/tools/cli-ref-push.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/donnie-msft-NuGetSkipDuplicates/docs/tools/cli-ref-push.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-push?branch=pr-en-us-1444) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201905160119155733-1444%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=40AAW%2BRL3IIg3K4oCxmgqBAsDn7s2qEtvbQM4LKt%2F%2Bk%3D&st=2019-05-16T01%3A15%3A25Z&se=2019-06-16T01%3A20%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tools/cli-ref-push.md [hunk 1]
```diff
@@ -63,4 +64,6 @@ nuget push foo.nupkg 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -Source https://api.nu
 nuget push foo.nupkg 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a
 
 nuget push foo.nupkg 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -src https://customsource/
+
+nuget push foo.nupkg 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -src https://customsource/ -SkipDuplicate
```
Comment (karann-msft) [inline docs/tools/cli-ref-push.md]:
```suggestion
:: In the example below -SkipDuplicate will skip pushing the package if package "Foo" version "5.0.2" already exists on NuGet.org
nuget push Foo.5.0.2.nupkg 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -src https://api.nuget.org/v3/index.json -SkipDuplicate
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[cf04c79](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1444/commits/cf04c79ecf3088229001a8752ac133377c6f3618)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/tools/cli-ref-push.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/donnie-msft-NuGetSkipDuplicates/docs/tools/cli-ref-push.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-push?branch=pr-en-us-1444) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C21%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201905211607534898-1444%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=P4qb2%2BY%2F3cdFveXDCeNfkj2psuDSHUpUf1h8ffR4lzs%3D&st=2019-05-21T16%3A04%3A00Z&se=2019-06-21T16%3A09%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tools/cli-ref-push.md (hunk 1 above)
Comment (donnie-msft) [reply docs/tools/cli-ref-push.md]: @karann-msft  Did you expect there to be two colons "::" on the front of this line?

**Pull request merged**